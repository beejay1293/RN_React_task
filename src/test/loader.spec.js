import React from 'react';
import { shallow } from 'enzyme';
import  Loaders from '../components/Dashboard/loader';

describe('test Header component', () => {
  it('should render component', () => {
    const Loader = shallow(<Loaders  />);
    expect(Loader).toMatchSnapshot();
  });
});
