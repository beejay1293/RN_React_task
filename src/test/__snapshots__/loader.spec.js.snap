// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Header component should render component 1`] = `
<div
  className="Loader"
>
  <Loader
    className=""
    color="#ffaf30"
    height={100}
    style={Object {}}
    timeout={0}
    type="Circles"
    visible={true}
    width={100}
  />
</div>
`;
